test/sanity/code-smell/package-data.requirements.txt

```diff
@@ -11,7 +11,6 @@ resolvelib==1.0.1
 rstcheck==5.0.0
 semantic-version==2.10.0
 setuptools==66.1.0
-tomli==2.0.1
```

Comment:
It looks like the requirements were updated using Python 3.11 or 3.12, since `tomli` is omitted. Make sure to run the update script using Python 3.10, since that's our minimum supported version.